### PR TITLE
P.nhan

### PR DESCRIPTION
**Pull request by LePhuNhan**

### CODE REVIEW

**Review by Nhan-To — APPROVED** [review]
Looks good!

File: backend/index.js
Comment (duyngodev) [inline backend/index.js:1]: Already config cors from line 13 - 18

File: backend/package.json
Comment (duyngodev) [inline backend/package.json:1]: We use Mongoose for cloud server, so we don't need Mongodb any more

File: frontend/src/components/nhandev/new.jsx
Comment (duyngodev) [inline frontend/src/components/nhandev/new.jsx:1]: Delete this

**Review by duyngodev — APPROVED** [review]
Delete second cors in index.js (backemd)
Delete mongodb for Atlas (Back end)
Two getAlll function() in product.controller.js

**Pull request merged**